charts/postgres/README.md

```diff
@@ -121,6 +121,7 @@ helm uninstall my-release
 | customConfig | string | `nil` | Optional custom configuration block that will be mounted as file in `/etc/postgresql/postgresql.conf` |
 | extraEnvSecrets | list | `[]` | A list of existing secrets that will be mounted into the container as environment variables |
 | extraSecretConfigs | string | `nil` | An existing secret with files that will be added to the postgres configuration in addition to `/etc/postgresql/postgresql.conf` |
+| customScripts | object | `nil` | Optional custom scripts that can be defined inline and will be mounted as files in `/docker-entrypoint-initdb.d` |
```

Comment:
I wonder if we should rename the existing `extraScripts` to `extraScriptsConfigMap` and call this new one just `extraScripts`.

It would be a breaking change, but I think it would be more intuitive.

Or, if you really want to avoid the breaking change, perhaps a better name would be `extraScriptsInline`.

Reply:
> mm.. all the extra*-stuff in all my charts are always mounted configMaps or Secrets and the custom* parts are inline configuration. Thats why I used that name.

Fair enough.

> But to make it more clear, I can rename it to "customInlineScripts:"

Don't bother changing the name if the current one matches your pattern.

Reply:
I adapted it to `customInlineScripts:` and v0.4.0

Reply:
I checked some other charts you have, like the mongodb one, and it accepts a `customConfig` parameter, which is kinda similar to this one. Since you don't call it `customInlineConfig` there, perhaps you shouldn't call `customInlineScripts` here too (and revert to `customScripts`), for consistency.

What do you think?

Reply:
yep.. Thats correct. I will revert it back to customScripts.
When the customScripts: feature works for you, I will also add this feature to mongodb, mariadb, mysql